### DISCUSSION TITLE
SuperSlicer not opening (at all)

### DISCUSSION THREAD

**Discussion by EricB420** [supermerill/SuperSlicer#2887] [Q&A]
Hello everyone,

I am just starting with Superslicer, and downloaded the latest Gravel solid version [2.3.57.12] for Windows. All went well except that when I double click on superslicer.exe nothing happens (at all) : no flash screen, no process running in the background, nothing. 

I am running Win10 on a i7 6500U laptop (integrated HD Graphics 520 integrated). my OpenGL version : 4.5

Eric.

**Comment by besser** [comment]
First, force to close SuperSlicer and then remove the `SuperSlicer.ini` and `SuperSlicer.ini.bak` files and `cache` and `ui_layout` folders at `C:\Users\<yourusername>\AppData\Roaming\SuperSlicer`.

If not work, try to remove the entirely SuperSlicer folder: `C:\Users\<yourusername>\AppData\Roaming\SuperSlicer`

Work for me.

**Reply by EricB420** [reply]
Hello Roberto. Thank you for your help !!
I checked in \AppData\Roaming\ but there isn't any SuperSlicer folder. While this is kind of expected since this is the first time I try to use SuperSlicer, could this be the cause of my issue?

**Reply by EricB420** [reply]
By the way I also cannot open the superslicer_console.exe application : the console opens for 1s or so before it disappears.
Interestingly, I have exactly the same issue with PrusaSlicer.

**Comment by EricB420** [comment]
Hurray I got it working !!
The issue was linked with my Intel Integrated Graphics card. Even though the drivers were all up to date, it was not working. I disabled the internal graphics card in Windows device manager and ran out of my AMD Radeon graphics card. This fixed it.

Thank you for your help in any case :-)

Edit: while disabling my integrated graphics card worked, I do not want to disable it permanently either. I assigned SuperSlicer so that it runs off my Radeon in the Radeon settings, but that did not work. Interestingly, if I open SuperSlicer with Integrated graphics card disabled, and then enables again the card SuperSlicer continues running fine. So it seems to be linked with the way SuperSlicer opens ... Any thoughts ?
